Lib/test/test_platform.py

```diff
@@ -741,5 +746,78 @@ def test_parse_os_release(self):
         self.assertEqual(len(info["SPECIALS"]), 5)
 
 
+class CommandLineTest(unittest.TestCase):
+    def setUp(self):
+        self.clear_caches()
+        self.addCleanup(self.clear_caches)
+
+    def clear_caches(self):
+        platform._platform_cache.clear()
+        platform._sys_version_cache.clear()
+        platform._uname_cache = None
+        platform._os_release_cache = None
+
+    @staticmethod
+    def text_normalize(string):
+        """Dedent *string* and strip it from its surrounding whitespaces.
+        This method is used by the other utility functions so that any
+        string to write or to match against can be freely indented.
+        """
```

Comment:
```suggestion
        """Dedent *string* and strip it from its surrounding whitespaces.

        This method is used by the other utility functions so that any
        string to write or to match against can be freely indented.
        """
```